### PR TITLE
Feature/wiremock

### PR DESCRIPTION
**Pull request by dwybourn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,8 +24,10 @@
     "test:compatibility:local:test": "mkdir -p build/reports/compatibility && cross-env TESTS=localCompatibility cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config' -f json:build/reports/compatibility-report.json",
     "test:compatibility:local:report": "node src/test/compatibility/local-compatibility-report.js",
     "test:accessibility": "node 'src/test/a11y/test-suite.js'",
-    "test:browser": "npm run test:wait && npm-run-all -s test:smoke test:accessibility test:compatibility:local test:acceptance test:acceptance:report",
+    "test:browser": "npm run test:wait && npm run test:startWiremock && npm-run-all -s test:smoke test:accessibility test:compatibility:local test:acceptance test:acceptance:report test:stopWiremock",
```
Comment (gethyn1) [inline package.json]: Could maybe use `npm-run-all` for the first two commands as well here:
```
"test:browser": "npm-run-all -s test:wait test:startWiremock && npm-run-all -s test:smoke test:accessibility test:compatibility:local test:acceptance test:acceptance:report test:stopWiremock"
```

Comment (dwybourn) [reply package.json]: Updated to use npm run all.

File: src/test/a11y/test-suite.js [hunk 2]
```diff
@@ -23,12 +24,25 @@ const dateIn3Months = () => {
...
+  }`)
+}
+
 /*
   Runs though the application, evaluating each page and performing post requests to populate the necessary
   data in the database.
  */
 const runTests = async () => {
   try {
+    postMappingToWiremock()
```
Comment (gethyn1) [inline src/test/a11y/test-suite.js]: I would maybe have less abstraction here:
```
postJsonData(WIREMOCK_MAPPING_URL, WIREMOCK_CLAIMS_SUCCESS)
```

Comment (dwybourn) [reply src/test/a11y/test-suite.js]: Done.

File: package.json [hunk 3]
```diff
@@ -56,6 +58,7 @@
     "cross-env": "^5.2.0",
     "cucumber": "^5.1.0",
     "cucumber-html-reporter": "^4.0.4",
+    "download": "^7.1.0",
```
Comment (gethyn1) [inline package.json]: What does `download` do? I can't see it used anywhere.

Comment (dwybourn) [reply package.json]: Left in by mistake, now removed.

**Pull request merged**